tests/lct_test.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-u"""Tests for linear canonical transform functions
+"""Tests for linear canonical transform functions
```

Comment:
Use our normal header here.

Reply:
This wasn't done.

Reply:
like with 
```python
u""" 
...
:copyright: Copyright (c) 2019 RadiaSoft LLC.  All Rights Reserved.
:license: http://www.apache.org/licenses/LICENSE-2.0.html
...
"""
```
?

Reply:
Yes but don't need `u"""`  just `"""`. Describe what the file does on the first line.